dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/MetadataDumper.java

```diff
@@ -279,7 +279,7 @@ public <T> T runChildTask(Task<T> task) throws MetadataDumperUsageException {
         TASK:
         for (int i = 0; i < tasks.size(); i++) {
           runTask(runContext, state, tasks.get(i));
-          log("Finished " + (i + 1) + " out of " + tasks.size() + " tasks");
+          LOG.info("Finished " + (i + 1) + " out of " + tasks.size() + " tasks");
```

Comment:
I would suggest to use a separate slf4j logger to log progress, so that it does not get mixed up with other logs. Appenders are by default cumulative, but you can override this with `additivity` attribute.

https://logback.qos.ch/manual/configuration.html#overridingCumulativity

Reply:
Done